tests/integration/linux_bridge_test.py

```diff
@@ -397,6 +397,131 @@ def test_port_vlan_not_implemented(port0_up):
         pass
 
 
+@pytest.mark.xfail(
+    reason="https://nmstate.atlassian.net/browse/NMSTATE-230", strict=True
+)
+class TestVlanFiltering:
+    access_tag = 300
+    bridge_name = TEST_BRIDGE0
+
+    @pytest.mark.parametrize(
+        "is_native_vlan", [True, False], ids=["native", "not-native"]
+    )
+    def test_trunk_port_config(self, is_native_vlan, port0_up):
+        trunk_tags = TestVlanFiltering._generate_vlan_id_config(100, 101)
+        trunk_port_state = TestVlanFiltering._generate_vlan_filtering_config(
+            "trunk",
```

Comment:
Seems like this should come from the schema.

Reply:
Done